src/sentry/options/defaults.py

```diff
@@ -1565,6 +1565,23 @@
 # Killswitch for monitor check-ins
 register("crons.organization.disable-check-in", type=Sequence, default=[])
 
+# Globally enables the check_accept_monitor_checkin method to be run during
+# monitor check-ins. This is temporarily in support of billing in getsentry.
+register(
+    "crons.check-accept-monitor-checkin-enabled",
+    default=False,
+    type=Bool,
+    flags=FLAG_ALLOW_EMPTY | FLAG_AUTOMATOR_MODIFIABLE,
+)
+
+# A list of monitor slugs that should have the check_accept_monitor_checkin
+# method run, even when check-accept-monitor-checkin-enabled is False.
+register(
+    "crons.check-accept-monitor-checkin-slug-overrides",
```

Comment:
is the purpose of this flag for testing the consumer logic before we flip the above flag on?

Reply:
yeah correct